android/app/build.gradle

```
@@ -8,7 +8,8 @@ if (localPropertiesFile.exists()) {
 
 def flutterRoot = localProperties.getProperty('flutter.sdk')
 if (flutterRoot == null) {
-    throw new GradleException("Flutter SDK not found. Define location with flutter.sdk in the local.properties file.")
+   throw new GradleException("Flutter SDK not found. Define location with flutter.sdk in the local.properties file.")
```

Are the following changes automatically generated or manually changed? If manually changed, please remove unnecessary changes.

---

had a little error with this but found out afterwards that it's safe to ignore the error as it's vs code related or something and we shouldn't mess with the way it throws exception, will rectify that